x/stakeibc/keeper/reward_converter.go

```diff
@@ -51,6 +57,119 @@ type ForwardMetadata struct {
 // and the normal staking and distribution flow will continue from there.
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+// Breaks down the split of non-native-denom rewards into the portions intended for a rebate vs the remainder
+// that's used for fees and reinvestment
+// For instance, in the case of dYdX, this is used on the USDC that has not been pushed through the trade route
+// yet, and has not yet been converted to DYDX
+//
+// The rebate percentage is determined by: (% of total TVL contributed by commuity pool) * (rebate percentage)
+//
+// E.g. Community pool liquid staked 1M, TVL is 10M, rebate is 50%
+// Total rewards this epoch are 1000, and the stride fee is 10%
+// => Then the rebate is 1000 rewards * 10% stride fee * (1M / 10M) * 50% rebate = 5
+func (k Keeper) CalculateRewardsSplitRewardDenom(
```

Comment:
Sorry to be pedantic about this: `RewardDenom` is confusing to me, afaiu we'd only call this function on chains who have rewards in non-host denom (non-native denoms). Why not just call it that? RewardDenom is confusing because the denom of rewards is usually the native denom, but this function should never be run on native denoms.

Reply:
I have the same comment for the naming of `CalculateRewardsSplitRewardDenom`

Reply:
as mentioned before the RewardDenom/HostDenom split is used across the codebase for the trade route. 

I'm open to renaming it, but I'd prefer to avoid a refactor of all the trade route stuff and I think having inconsistent terminology across the codebase is the worst thing we can do. 

What about `CalculateRewardsSplitBeforeRebate` and `CalculateRewardSplitAfterRebate`?

Reply:
`CalculateRewardsSplitBeforeRebate` and `CalculateRewardSplitAfterRebate` is much more clear, in favor!

Agreed that a large renaming now is tricky, sorry had forgotten about that detail.

Reply:
pushed!